fail2ban/client/jailreader.py

```diff
@@ -83,13 +84,21 @@ def isEnabled(self):
 			self.__opts and self.__opts.get("enabled", False))
 
 	@staticmethod
-	def _glob(path):
-		"""Given a path for glob return list of files to be passed to server.
+	def _glob(path, excludeGlobs):
```

Comment:
small change for backwards compatibility (+ optional, little performance fix by too many entries)
```diff
-	def _glob(path, excludeGlobs):
+	def _glob(path, excludeGlobs=[]):
...
		pathList = []
+		excludeGlobs = set(excludeGlobs)
		for p in glob.glob(path):
```